src/Compiler/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorCSharpDocument.cs

```diff
@@ -41,15 +42,15 @@ internal class DefaultRazorCSharpDocument : RazorCSharpDocument
         _options = options;
 
         _diagnostics = diagnostics ?? Array.Empty<RazorDiagnostic>();
-        _sourceMappings = sourceMappings ?? Array.Empty<SourceMapping>();
+        _sourceMappings = sourceMappings;
```

Comment:
Consider calling `NullToEmpty()` since there was previously a null check.

Reply:
The only null I found was the change immediately above this, which I changed to Empty. My assumption is this is a safe change because the compiler is no longer public API, right?

Reply:
Sounds good to me.